packages/blocks/src/api/utils.js

```diff
@@ -121,3 +123,105 @@ export function normalizeBlockType( blockTypeOrName ) {
 
 	return blockTypeOrName;
 }
+
+/**
+ * Get the label for the block, usually this is either the block title,
+ * or the value of the block's `label` function when that's specified.
+ *
+ * @param {Object} blockType  The block type.
+ * @param {Object} attributes The values of the block's attributes.
+ * @param {Object} context    The intended use for the label.
+ *
+ * @return {string} The block label.
+ */
+export function getBlockLabel( blockType, attributes, context = 'visual' ) {
+	const {
+		__experimentalLabel: getLabel,
```

Comment:
Was it intentional that this was called `__experimentalLabel` and not `__experimentalGetLabel`, as the deconstructed assignment would imply that its stable form might take?

I would expect either one of the following:

```suggestion
		__experimentalGetLabel: getLabel,
```

```suggestion
		__experimentalLabel: label,
```

While I'd grant that most functions of a block type don't have these sorts of prefixes ("get"), the reason for that as I'd see it is that they're all (to my knowledge) already in verb forms anyways: "save", "edit", etc.

Reply:
👍 Yep, I wasn't sure about this, thanks for clarifying.